qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -766,10 +769,11 @@ def save(self) -> None:
                 self._service.delete_figure(experiment_id=self.experiment_id, figure_name=name)
             self._deleted_figures.remove(name)
 
-        print(
-            "You can view the experiment online at https://quantum-computing.ibm.com/experiments/"
-            + self.experiment_id
-        )
+        if verbose:
+            print(
```

Comment:
@coruscating What do you think?

Reply:
Actually probably best to leave as print, I find it quite useful since you can click on the link when it prints in jupyter. I just don't really like adding `verbose` flags to methods. Usually better to have some class attribute or something that could be set in 1 place rather than every time you call the method for these sort of things.